test/test_shared.js

```
@@ -31,7 +31,9 @@ const dom = new JSDOM('<!DOCTYPE html><html><body></body></html>', {
 });
 
 // Make window and document available globally
+// eslint-disable-next-line no-global-assign
 window = dom.window;
```

Biome (2.1.2)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)